crates/task-impls/src/events.rs

```diff
@@ -28,6 +29,24 @@ impl<TYPES: NodeType> TaskEvent for HotShotEvent<TYPES> {
     }
 }
 
+/// Wrapper type for the event to notify tasks that a proposal for a view is missing
+/// and the channel to send the event back to
+#[derive(Debug, Clone)]
+pub struct ProposalMissing<TYPES: NodeType> {
```

Comment:
Would a better name be something like `QuorumProposalRequest`? (Since it has the channel in it)